docs/source/installation/development.rst

```diff
@@ -3,5 +3,268 @@
+    pip install virtualenv
+
+
+Creating the directory structure
+--------------------------------
+
+You will need a directory in your file system to store Indico as well as its data files (archives, etc...). Some
+developers keep all their code inside a ``dev`` or ``code`` dir. We will assume ``dev`` here.
+
+.. code-block:: shell
+
+    mkdir -p ~/dev/indico/data
+
+We will need a virtualenv where to run Indico::
+
+    cd ~/dev/indico
+    virtualenv env -p /usr/bin/python2.7
+
+
+Cloning Indico
+--------------
+
+First, let's clone Indico's code base. If you're going to contribute back to the project, it's probably best if you
+clone your own `GitHub fork of the project <https://help.github.com/articles/fork-a-repo/>`_ and set it as the origin::
+
+    git clone --recursive git@github.com:<your-github-username>/indico.git src
+    cd src
+    git remote add upstream https://github.com/indico/indico.git src
+    cd ..
+
+Otherwise, cloning the upstream repository as the origin should be enough::
+
+    git clone --recursive https://github.com/indico/indico.git src
+
+If you're going to be changing the standard Indico plugins and/or the documentation, you can also clone those::
+
+    mkdir plugins
+    git clone https://github.com/indico/indico-plugins.git plugins/base
+    git clone https://github.com/indico/indico-user-docs.git user-docs
+
+
+Setting up Maildump (recommended)
+---------------------------------
+
+Some actions in Indico trigger automatic e-mails. Those will normally have to be routed through an SMTP server.
+This can become a problem if you're using production data and/or real e-mails, as users may end up being spammed
+unnecessarily. This is why we advise that you include a fake SMTP server in your development setup.
+`Maildump <https://github.com/ThiefMaster/maildump>`_ does exactly this and runs on Python. It should be quite simple
+to set up::
+
+    virtualenv maildump -p /usr/bin/python2.7
+    ./maildump/bin/pip install maildump
+    ./maildump/bin/maildump -p /tmp/maildump.pid
+
+You'll then be able to access the message log at `<http://localhost:1080>`_.
+
+
+Creating the DB
+---------------
+
+.. code-block:: shell
+
+    sudo -u postgres createuser <your-user> --createdb
+    sudo -u postgres createdb indico_template -O <your-user>
+    sudo -u postgres psql indico_template -c "CREATE EXTENSION unaccent; CREATE EXTENSION pg_trgm;"
+    createdb -O <your-user> indico -T indico_template
+
+
+Configuring
+-----------
+
+Let's get into the Indico virtualenv::
+
+    source ./env/bin/activate
+
+    cd src
+    pip install -r requirements.dev.txt
+    pip install -e .
+    fab setup_deps:system_node=1
+
+Then, follow the instructions given by the wizard::
+
+    indico setup wizard --dev
+
+You can then initialize the DB::
+
+    indico db prepare
+
+
+Running Indico
+--------------
+
+.. code-block:: shell
+
+    indico run -h <your-hostname> -q --enable-evalex
```

Comment:
`-h 127.0.0.1` since that's what we use in the default base url generated by the wizard?

Reply:
But the user is still free to choose whatever they want, so why confuse them?

Reply:
true, the setting here overrides `BASE_URL` anyway...